### PR TITLE
save client version

### LINKED ISSUE #3479: The nightly installation package displays the wrong version
image

### PR DESCRIPTION
**Pull request by heroicNeZha**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What does this PR do?
There are two versions here. 
One is the protocol version in thrift, which is used to verify compatibility through the whitelist; 
the other is the packaged version, which is tag or datetime-nightly.

+-------------+-------+----------+-----------+--------------+----------------+
 | Host               | Port     | Status      | Role            | Git Info Sha     | Version               |
+-------------+-------+----------+-----------+--------------+----------------+
 | "127.0.0.1"     | 21079 | "ONLINE" | "STORAGE" | "75f11121"    | "datetime-nightly" |
+-------------+-------+----------+-----------+--------------+----------------+

#### Which issue(s)/PR(s) this PR relates to?
closes #3479 
  
#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: src/interface/meta.thrift [hunk 1]
```diff
@@ -1123,8 +1123,9 @@ struct VerifyClientVersionResp {
 
 
 struct VerifyClientVersionReq {
-    1: required binary version = common.version;
-    2: common.HostAddr host;
+    1: required binary thrift_version = common.version;
+    2: binary build_version;
+    3: common.HostAddr host;
```
Comment (Shylock-Hg) [inline src/interface/meta.thrift:1128]: Don't modify the field number, you could add it at end.

Comment (heroicNeZha) [reply src/interface/meta.thrift]: done

File: src/interface/meta.thrift [hunk 2]
```diff
@@ -1123,8 +1123,9 @@ struct VerifyClientVersionResp {
 
 
 struct VerifyClientVersionReq {
-    1: required binary version = common.version;
+    1: required binary thrift_version = common.version;
```
Comment (Shylock-Hg) [inline src/interface/meta.thrift:1126]: `client_version` is better.

**Review by Shylock-Hg — APPROVED** [review]

File: src/interface/meta.thrift [hunk 3]
```diff
@@ -1123,8 +1123,9 @@ struct VerifyClientVersionResp {
 
 
 struct VerifyClientVersionReq {
-    1: required binary version = common.version;
+    1: required binary client_version = common.version;
     2: common.HostAddr host;
+    3: binary build_version;
```
Comment (HarrisChu) [inline src/interface/meta.thrift:1128]: why need build_version?
if the thrift versions are in the same, client could connect with server too.

Comment (heroicNeZha) [reply src/interface/meta.thrift:1128]: Not For verify，Just for `show hosts` to show version

**Review by Aiee — APPROVED** [review]

**Pull request merged**